webwindow.py

```diff
@@ -168,6 +169,18 @@
         F24: 'f24',
     };
 
+    const canvasRect = canvas.getBoundingClientRect();
```

Comment:
move this back in, no need to optimize the # of calls to it, what if something resizes the canvas?